pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -588,6 +588,60 @@ var _ = Describe("Replace Nodes", func() {
 		Expect(cloudProvider.CreateCalls).To(HaveLen(0))
 		ExpectNodeExists(ctx, env.Client, node.Name)
 	})
+	It("can replace node for expiration", func() {
```

Comment:
Do we have checks here where expiration triggers a multi-node creation since that scenario is now being added in this PR?

Reply:
Added it in.